SECP-service/src/main/resources/assets/app/views/profile/group-profile.html

```diff
@@ -0,0 +1,31 @@
+<div>
+    <div id="groupprofile">
+        <div>
+            <img src="https://scontent.fykz1-1.fna.fbcdn.net/v/t1.0-1/c90.210.540.540/s160x160/19510125_10213767937303246_5095195573619394298_n.jpg?oh=7768b901dcef45df1ccfa57878be3e89&oe=5AC0FE69"
+                class="img-responsive img-circle">
+        </div>
+        <h4 id="group-name">Group Name</h4>
+        <div id="user-list">
+            <h3 class="list-heading">Group Members</h3>
+            <ul class="list-group" id="group-members">
```

Comment:
I see that you are adding each user to the ul. You need to wrap this in a scrollbar because if there are 100 users. Then the page would be too long.